include/exadg/functions_and_boundary_conditions/interface_coupling.h

```diff
@@ -232,20 +249,24 @@ class InterfaceCoupling
     for(auto quadrature : interface_data_dst->get_quad_indices())
     {
 #if DEAL_II_VERSION_GTE(10, 0, 0)
-      std::vector<dealii::Tensor<rank, dim, Number>> const result =
+      auto const result =
```

Comment:
@kronbichler Wanted me to be explicit here due to the possible double/Number confusion.

Reply:
this is currently the strategy to deal with https://github.com/dealii/dealii/issues/13517 in the case of scalar quantities. Of course, we still need the copying some lines below.